js/__internal/grids/grid_core/module_utils.ts

```diff
@@ -696,4 +696,44 @@ export default {
 
     return lookupDataSource;
   },
+
+  logHeaderFilterDeprecatedWarningIfNeed(component) {
```

Comment:
Minor: Extract logHeaderFilterDeprecatedWarningIfNeed to the separated file.

Reply:
I think it is redundant